<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test5Dialog</class>
 <widget class="QDialog" name="test5Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>液氮泵</string>
  </property>
  <property name="windowIcon">
   <iconset resource="errordetection.qrc">
    <normaloff>:/images/images/timg.jpeg</normaloff>:/images/images/timg.jpeg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>52</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>液氮泵电压</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test52">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test52">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>液氮泵电流</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test51">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test51">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>49</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>液氮泵功率</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test53">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test53">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="errordetection.qrc"/>
 </resources>
 <connections/>
</ui>
